hypothesis-python/src/hypothesis/strategies/_internal/core.py

```diff
@@ -988,7 +994,7 @@ def builds(
 
 @cacheable
 @defines_strategy(never_lazy=True)
-def from_type(thing: Type[Ex], *, force_defer: bool = False) -> SearchStrategy[Ex]:
```

Comment:
Really unrelated, but it was indicated that you didn't want to change the public API?